src/util/taskManager.test.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+import {TaskManager} from "./taskManager";
+
+import {SilentTaskReporter} from "./taskReporter";
+
+describe("util/taskManager", () => {
+  class TestCase {
+    taskManager: TaskManager;
+    reporter: SilentTaskReporter;
+    constructor() {
+      this.reporter = new SilentTaskReporter();
+      this.taskManager = new TaskManager(this.reporter);
+    }
+    start(task: string): TaskManager {
+      return this.taskManager.start(task);
+    }
+    finish(task: string) {
+      this.taskManager.finish(task);
+      return this;
+    }
+  }
```

Comment:
personally, i would probably have just written something like

```js
function example() {
  const reporter = new SilentTaskReporter();
  const manager = new TaskManager(reporter);
  return {manager, reporter};
```

Your way is fine and maybe better, no change requested.